R/ResampleDesc.R

```diff
@@ -141,6 +149,53 @@ makeResampleDescRepCV = function(reps = 10L, folds = 10L) {
   makeResampleDescInternal("repeated cross-validation", iters = folds * reps, folds = folds, reps = reps)
 }
 
+makeResampleDescFixedCV = function(horizon = 1L, initial.window = .5, size = 100L, skip = .01  ) {
+  horizon = asInteger(horizon, lower = 1L, upper = Inf)
+  assertNumeric(initial.window, lower = 1E-15, upper = 1)
+  size = asInteger(size, lower = 1L, upper = Inf)
+  assertNumeric(skip, lower = 0L, upper = Inf)
+  initial.window.desc = floor(initial.window * size)
+  skip.desc = floor(skip * size)
+  stops = seq(from = 1, to = size)[initial.window.desc:I(size - horizon)]
+  starts = stops - initial.window.desc + 1
+  thin = function(x, skip = 2) {
+    n = length(x)
+    x[seq(1, n, by = skip.desc)]
+  }
+  if (skip > 0)
+    iters = length(thin(starts, skip = skip + 1))
+  else
+    iters = length(starts)
+
+  makeResampleDescInternal("fixed", iters = iters,  horizon = horizon,
+    initial.window = initial.window, skip = skip,
+    initial.window.desc = initial.window.desc, skip.desc = skip.desc)
+}
+
+makeResampleDescGrowingCV = function(horizon = 1L, initial.window = 10L, size = 100L, skip = 0L) {
```

Comment:
Remove `size`, see comments above.

Also `skip` default is inconsistent with `FixedCV`.

`initial.window` default is inconsistent with `FixedCV`. Also, further down, `assertNumeric(initial.window, upper=1)` will complain about it being `> 1`.